src/app/core/components/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,8 +1,18 @@
 import { Component } from '@angular/core';
 
+import { Observable } from 'rxjs';
+
+import { AuthService } from '@app/services/auth/auth.service';
+
 @Component({
   selector: 'app-breadcrumbs',
   templateUrl: './breadcrumbs.component.html',
   styleUrls: [ './breadcrumbs.component.scss' ],
 })
-export class BreadcrumbsComponent { }
+export class BreadcrumbsComponent {
+  constructor(private authService: AuthService) { }
+
+  get showComponent$(): Observable<boolean> {
```

Comment:
I'd propose using an additional application template for login page. I can demonstrate it on Q&A session. if you would like.

Reply:
here is actually no need to tie dummy BreadcrumbsComponent with authorization logic

Reply:
Yes, let's have additional application template for login page demonstrated on Q&A session. Cause I'm not completely understand what do you mean by it. Regarding the logic in BreadcrumbsComponent - I'll refactor it.

Reply:
Yes, let's have additional application template for login page demonstrated on Q&A session. Cause I'm not completely understand what do you mean by it. Regarding the logic in BreadcrumbsComponent - I'll refactor it.